src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -28,3 +28,77 @@
 .row .edu-intro-spacing {
   margin-bottom: 7rem;
 }
+
+// autocomplete component
+.ds-c-autocomplete__list {
```

Comment:
It would be nice to take advantage of Sass's `&` on these selectors. `.ds-c-autocomplete__list` could be at the top level, and nested in that could be `&-item {...}` and in that `&--item`. And so on. [This post](http://alwaystwisted.com/articles/2014-02-27-even-easier-bem-ing-with-sass-33) explains what I'm talking about if I'm not being clear.

Reply:
Thanks- I suck at css so l appreciate the link.